### PR TITLE
[16.0][IMP] ddmrp_adjustment: give more visibility

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
When an ADU is affected by a DAF, it will appear a button next to the ADU to check which are being applied and what is the factor.

@ForgeFlow

### CODE REVIEW

File: ddmrp_adjustment/models/stock_buffer.py [hunk 1]
```diff
@@ -22,6 +22,13 @@ class StockBuffer(models.Model):
         "parent buffers.",
     )
 
+    future_daf_chart = fields.Text(
+        string="Future DAF Chart",
+    )
```
Comment (LoisRForgeFlow) [inline ddmrp_adjustment/models/stock_buffer.py]: Remove this

File: ddmrp_adjustment/models/stock_buffer.py [hunk 2]
```diff
@@ -176,3 +184,13 @@ def action_view_demand_to_components(self):
             "view_mode": "tree",
             "domain": [("id", "in", demand_ids)],
         }
+
+    def action_view_affecting_adu(self):
+        demand_ids = (
+            self.env["ddmrp.adjustment"].search(self._daf_to_apply_domain()).ids
```
Comment (LoisRForgeFlow) [inline ddmrp_adjustment/models/stock_buffer.py]: Can you show all future adjustments that apply and will apply? 

You can also add a filter, "current" that is added by default.

But the ability to remove that default filter and see future adjustment is valuable I think.

**Review by LoisRForgeFlow — COMMENTED** [review]
Some comments below

File: ddmrp_adjustment/models/stock_buffer.py (hunk 2 above)
Comment (DavidJForgeFlow) [reply ddmrp_adjustment/models/stock_buffer.py]: Done!

**Pull request merged**